packages/api/src/router/app.ts

```diff
@@ -111,16 +111,19 @@ export const appRouter = createTRPCRouter({
   create: permissionRequiredProcedure
     .requiresPermission("app-create")
     .input(validation.app.manage)
-    .output(z.void())
+    .output(z.object({ appId: z.string() }))
```

Comment:
Does it make sense to name it appId? I would say this is unnecessary

Reply:
In my opinion yes, because it is not clear from the ``create`` verb what this ID is used for

Reply:
IMO it is already known in the contexts that it is used, but let's leave it like that